srcpkgs/wlrctl/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'wlrctl'
+pkgname=wlrctl
+version=0.2.1
+revision=1
+wrksrc="wlrctl-v0.2.1"
```

Comment:
you didn't save your changes :smile:

Reply:
Fixed.

Also, please let me know if there's a policy against people using a noreply.github.com email in the repos.

Reply:
>policy against people using a noreply.github.com email in the repos
yes: https://github.com/leahneukirchen/xtools/blob/master/xlint#L319

Reply:
Fine. I've added an email of mine.